kernelci/build.py

```diff
@@ -435,7 +435,7 @@ def _output_to_file(cmd, log_file, rel_dir=None):
 
 def _run_make(kdir, arch, target=None, jopt=None, silent=True, cc='gcc',
               cross_compile=None, use_ccache=None, output=None, log_file=None,
-              opts=None, cross_compile_compat=None):
+              opts=None, cross_compile_compat=None, kselftest_kdir=None):
```

Comment:
I believe we can try to build with the output set to the local directory, like "normal" builds since this is what some kselftests expect. It's better to do that by changing the `build.jpl` Jenkins job rather than adding an extra argument with more complexity in the code.

Reply:
good input! I will update the code there. You have more xp in kernel-ci than me, when we know what the problem is, you and others can suggest other ways to solve, etc! Thanks @gctucker :)